### ISSUE TITLE
Fix Mobs spawn rate

### ISSUE THREAD

**Issue by geckosavage** [Tslat/Advent-Of-Ascension#2720]
**Mod Version**
3.5.4

**Describe the bug**
- Pixon don't spawn
- Some Aggressive mobs didn't spawn at all
- Incompatible w/ some biomes mod (BYG, BOP,...)

**Comment by Tslat** [comment]
Need more info than 'it doesn't work'.

There's a lot of aggressive mobs. Which ones don't work?
What makes you think it doesn't work with BYG/BOP?

**Comment by odw500** [comment]
From personal experience I've seen pixons spawning in dimensions that they weren't spawning in before version 3.5.4, but they spawn even more rarely now.

I've spent a few hours total in Runandor on my survival world and have only seen one Gleaming Pixon. I've travelled around the abyss for ~1 hour straight and have seen only three Shining Pixons. In L'borean on the other hand, Glaring Pixons spawn more frequently near me and in small groups at varying heights above ground level since it's underwater. Elsewhere, I haven't seen any since installing 3.5.4.
